Index.html

```diff
@@ -13,13 +13,19 @@
 <body>
   <header class="navbar">
     <a href="index.html" class="myLogo">My logo</a>

```

Comment:
[OPTIONAL] Make sure to make some adjustments also on the mobile version 👀 this is what I get for the Logo on Mobile 👇🏽 

![Screenshot 2023-05-07 at 01 35 42](https://user-images.githubusercontent.com/45739963/236659712-d4ae1ef6-72b0-4860-b589-952ed0086210.png)